gradle.properties

```
@@ -13,46 +13,26 @@
 # limitations under the License.
 
 group=org.ballerinalang
-version=0.4.1-SNAPSHOT
-ballerinaLangVersion=2201.8.2
+version=0.5.0-SNAPSHOT
+ballerinaLangVersion=2201.9.0-20240410-095500-2653a74d
```

```suggestion
ballerinaLangVersion=2201.9.0
```

Also change the ballerina version to `2201.9.0` in here: https://github.com/ballerina-platform/module-ballerinax-jaeger/blob/741c9328680e2f5fcf5943f80fa8d4160946bb6a/jaeger-extension-ballerina/Ballerina.toml#L21